modules/ocf/manifests/nginx_proxy.pp

```diff
@@ -46,19 +46,41 @@
         *                => $nginx_options;
 
       "${title}-redirect":
-        # We have to specify www_root even though we always redirect/proxy
-        www_root          => '/var/www',
-
         server_name       => concat([$server_name], $server_aliases),
-
         server_cfg_append => {
           'return' => "301 https://${server_name}\$request_uri"
         },
 
+        # We have to specify www_root even though we always redirect/proxy
```

Comment:
:(

Reply:
I actually haven't tried changing this, it's just been copied from place to place. I'll try and remove it and see if it still works, but I somehow doubt it will.

Reply:
Ah I see. Yeah I mostly was just marking my unhappiness at the situation. Thanks for trying without!

Reply:
So I tested this, and it worked without this specified, so that's pretty great! Thanks for the idea, I've removed it everywhere else this was specified and tested the redirects/proxying still works.